github/ContentFile.py

```diff
@@ -93,6 +93,18 @@ def html_url(self):
         self._completeIfNotSet(self._html_url)
         return self._html_url.value
 
+    @property
+    def last_modified(self):
+        """
+        :type: str
+        """
+        self._completeIfNotSet(self._last_modified)
+        commits = self.repository.get_commits(path=self.path)
+        self._last_modified = self._makeStringAttribute(
+            str(commits[0].commit.author.date)
```

Comment:
And this will result in an exception if any of those are not set.

Reply:
This is my reasoning for why all of these will always be set.. 
1. if self.path is valid, there has to be atleast one commit for the path to exist in the repo.
2. and every commit in git has to have an author. 

So maybe just checking for self.path should be sufficient here?

Reply:
Hmm.. As I am thinking about this, I can see its possible to create an instance with an invalid path, which would lead to commits being empty. I will add checks. Whats the expected return value in this case - `None`?